crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -409,6 +412,26 @@ impl MeshPipelineKey {
             _ => PrimitiveTopology::default(),
         }
     }
+
+    pub fn from_cull_mode(cull_mode: Option<Face>) -> Self {
+        let cull_mode = match cull_mode {
+            Some(Face::Back) => 0,
+            Some(Face::Front) => 1,
+            None => 2,
```

Comment:
Please make `None` be `0`, it feels more consistent and expected. And then it's a really minor thing but the wgpu enum has `Face::Front` first, and then `Face::Back` so I would follow that ordering.
```suggestion
            None => 0,
            Some(Face::Front) => 1,
            Some(Face::Back) => 2,
```